pandas/core/arrays/categorical.py

```diff
@@ -760,23 +760,32 @@ def categories(self) -> Index:
 
         Examples
         --------
+        For :class:`pandas.Series`:
 
-        For Series:
-
-        >>> ser = pd.Series(["a", "b", "c", "a"], dtype="category")
+        >>> ser = pd.Series(['a', 'b', 'c', 'a'], dtype='category')
         >>> ser.cat.categories
         Index(['a', 'b', 'c'], dtype='object')
 
-        >>> raw_cat = pd.Categorical(["a", "b", "c", "a"], categories=["b", "c", "d"],)
+        >>> raw_cat = pd.Categorical(['a', 'b', 'c', 'a'], categories=['b', 'c', 'd'])
```

Comment:
just for the future - in general, let's not bother with stylistic changes like changing double quotes to single quotes, at least until we have an automated tool to enforce this on docstrings